contrib/offline/redirect-port-80.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
+# Backup file
+if ! [ -f "/etc/hostapd/nat.sh.org" ]; then
+  sudo cp /etc/hostapd/nat.sh /etc/hostapd/nat.sh.org
```

Comment:
I think in other places in the codebase we have been using `.bak` for backup files